openapi/workday.expense/openapi.yml

```diff
@@ -0,0 +1,1534 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VALIDATION_ERROR_MODEL_REFERENCE'
+        default:
+          description: An error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ERROR_MODEL_REFERENCE'
+components:
+  schemas:
+    INSTANCE_MODEL_REFERENCE:
+      required:
+      - id
+      type: object
+      properties:
+        id:
+          pattern: ^(?:(?:[0-9a-f]{32})|(?:[0-9]+\$[0-9]+)|(\S+=\S+))$
+          type: string
+          description: wid / id / reference id
+        descriptor:
+          type: string
+          description: A description of the instance
+          readOnly: true
+        href:
+          type: string
+          description: A link to the instance
+          readOnly: true
+    ERROR_MODEL_REFERENCE:
+      required:
+      - error
+      type: object
+      properties:
+        error:
+          type: string
+          description: A description of the error
+    VALIDATION_ERROR_MODEL_REFERENCE:
+      allOf:
+      - $ref: '#/components/schemas/ERROR_MODEL_REFERENCE'
+      - type: object
+        properties:
+          errors:
+            type: array
+            description: An array of validation errors
+            items:
+              $ref: '#/components/schemas/ERROR_MODEL_REFERENCE'
+    FACETS_MODEL_REFERENCE:
+      type: array
+      items:
+        type: object
+        properties:
+          descriptor:
+            type: string
+            description: A description of the facet
+          facetParameter:
+            type: string
+            description: The alias used to select the facet
+          values:
+            type: array
+            description: the facet values
+            items:
+              required:
+              - id
+              type: object
+              properties:
+                count:
+                  type: integer
+                  description: The number of instances returned by this facet
+                  format: int32
+                id:
+                  pattern: ^(?:(?:[0-9a-f]{32})|(?:[0-9]+\$[0-9]+)|(\S+=\S+))$
+                  type: string
+                  description: wid / id / reference id
+                descriptor:
+                  type: string
+                  description: A description of the facet
+                href:
+                  type: string
+                  description: A link to the instance
+                  readOnly: true
+        description: This object represents the possible facets for this resource
+        readOnly: true
+    MULTIPLE_INSTANCE_MODEL_REFERENCE:
+      type: object
+      properties:
+        total:
+          type: integer
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/INSTANCE_MODEL_REFERENCE'
+    expenseItemsServiceRepresentation_978f4a3fe47110001a3243426e540060:
```

Comment:
Consider renaming the schema types although the official OpenAPI file have that name. The generated code is not user friendly